### DISCUSSION TITLE
Retention period for chats

### DISCUSSION THREAD

**Discussion by erickrf** [danny-avila/LibreChat#4067] [Feature Requests & Suggestions]
We are considering deploying LibreChat in my company, and for compliance reasons, we will most likely have to limit the retention period of chats stored in the mongo database.

So, basically, we would like to delete any chat history older than a certain time frame.

As far as I can tell, there is no such feature available. I suppose we could come up with some cronjob to perform this cleanup, but having this functionality integrated in the system would be helpful to other corporate users.

**Comment by dirkpetersen** [comment]
I also need such a feature, perhaps there is a workaround, e.g. purging mongodb records ?

**Comment by ryanrdetzel** [comment]
This could easily be a script that just deletes records in mongo older than X days.

**Comment by dirkpetersen** [comment]
See here : 

https://github.com/danny-avila/LibreChat/issues/2365#issuecomment-2378067337
